CHANGELOG.md

```diff
@@ -69,8 +69,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Changed command line interface.
   - Removed `cert`, `key`, `root` fields from config file.
   - Added cli options `-c`, `--cert`, `--key` and `--ca-certs`.
-- Changed `setGigantoConfig` to receive toml-string with full configuration.
-- Updated `gigantoConfig` to respond full configuration.
+- Renamed GraphQL API `gigantoConfig` to `config` and updated it to respond
+  the full configuration.
+- Changed the return type of the GraphQL API `setConfig` from `Result<String>`
+  to `Result<bool>`.
+- Updated `config` to respond full configuration.
```

Comment:
`setConfig` 에 대해서 API name change에 대한 내용이 빠져버렸습니다. 이 PR에서 변경된 사항 중 독자에게 알려야 하는 핵심이라서 빠지면 안 되는 것 같아요.

그리고 `config` GraphQL API 에 대한 설명은 1개의 항목으로 작성하는 것이 독자의 이해에 도움일 될 것 같습니다.

```suggestion
- Renamed GraphQL API `gigantoConfig` to `config` and updated it to respond the full configuration.
- Renamed GraphQL API `setGigantoConfig` to `setConfig`. The endpoint now accepts a full configuration as a TOML string and returns `Result<bool>`, instead of `Result<String>`.
```

위의 내용으로 적용 부탁드립니다.

Reply:
The change has been completed. 👍